SomosMasApp/SomosMasAppTests/SomosMasAppTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SomosMasAppTests.swift
+//  SomosMasAppTests
+//
+//  Created by Maximiliano Ituarte on 20/04/2022.
+//
+
+import XCTest
+@testable import SomosMasApp
+
+class SomosMasAppTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        // Any test you write for XCTest can be annotated as throws and async.
+        // Mark your test throws to produce an unexpected failure when your test encounters an uncaught error.
+        // Mark your test async to allow awaiting for asynchronous code to complete. Check the results with assertions afterwards.
+    }
+    
+    func testLoginDataInUserDefaults(){
+        
+        let storage = UserDefaults.standard
+        
+        let dictionary = storage.object(forKey: "Login-key") as? [String: String] ?? nil
+        
+        XCTAssertNotNil(dictionary)
```

Comment:
we talked about in the daily, test something more significant like save a change and bring it back testing if the key is the same, the text should be not repeatable like date or timestamp

Reply:
this test is a lazy test if you never logged in before it will fail